src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
I definitely prefer @mstriemer's API (`<Button alert small>Foo</Button>`)... I don't care if the internal implementation for the prop checking isn't as nice because we use that less than the component itself. The only benefit to `appearance` I can see is easily enforcing only one type of appearance mod... eg you can't have `appearance: 'small large light dark'`.

But then we can just do that with something like:

```js
if ([props.light, props.dark, props.small, props.large].reduce((prop) => return !!prop; )) {
  throw new Error('Conflicting style properties supplied to Button component');
}
```

Which I think is fine. If we can do that with Flow all the better...